Documentation/articles/getting_started/1_creating_a_new_project_vs.md

```diff
@@ -1,19 +1,75 @@
+
+To create new projects from within Visual Studio, you will need to install the Visual Studio 2019 extension which can be found in "Extensions -> Manage Extensions" in the Visual Studio menu bar.
+
+![Visual Studio Extension Manager](~/images/getting_started/1_VisualStudioExtensionManager.png)
+
+Once open, simply search for **MonoGame** in the top right search window (as shown above) and install the "MonoGame project templates".  You now have the MonoGame templates installed ready to create new projects.
+
+## MonoGame Content Pipeline Tool (MGCB Editor)
+
+The MGCB Editor which was previously available inside the MonoGame installer is now published to the .NET tools library. To install the MGCB Editor, simply open a command prompt and run the following dotnet command (if you get an error, please ensure you have installed the .NETCore SDK listed above):
+
+```
+dotnet tool install -g dotnet-mgcb-editor
+```
+
+> For development branches, you will need to ensure you have the dev channel NuGet registered and include the version number in the command, for example:
+>
+> ```
+> dotnet nuget add source -n MonoGame http://teamcity.monogame.net/guestAuth/app/nuget/feed/_Root/default/v3/index.json
+> 
+> dotnet tool install --global dotnet-mgcb-editor --version 3.8.0.1476-develop
+> ```
+
+Once installed, you also need to register the MGCB Editor with Visual Studio with the additional command from the command prompt:
+
+```
+mgcb-editor --register
+```
+
+For more details on the [MonoGame tools check here](~/articles/tools/tools.md).
+
+## Create Project
+
+Start Visual Studio and select **New Project...** in the upper left corner.
 
 ![New Solution](~/images/getting_started/1_new_soulution_vs.png)
 
-Now you should see a "New Project" dialog pop up, from here select **Templates > Visual C# > MonoGame** category, and then select **MonoGame Cross Platform Desktop Project**. Next type in the name that you wish to give your project, for this tutorial let's just use **ExampleGame** (do note that you should not use space character for it). After you've entered the name, click on the **Browse** button next to the location text field, and select where you wish to save your project. Finally click **OK** to create a new project.
+Now you should see a "New Project" dialog pop up, from here select **Templates > Visual C# > MonoGame** category, and then select **MonoGame Cross Platform Desktop Project**. Next type in the name that you wish to give your project, for this tutorial let's just use **ExampleGame** (*note* you should not use spaces when naming the project). After you've entered the name, click on the **Browse** button next to the location text field, and select where you wish to save your project. Finally click **OK** to create a new project.
```

Comment:
This is for VS2017. The new project wizard got an overhaul for VS2019.

Reply:
I'll double-check as I did update a lot of the assets for 2019/3.8